src/LogicalGraph.ts

```diff
@@ -918,6 +933,35 @@ export class LogicalGraph {
         return radius;
     }
 
+
+    getShortDescriptionBtnColor : ko.PureComputed<string> = ko.pureComputed(() => {
+        //this excludes graphs that have not been initiated by the user (eagle has an empty graph by default)
+        if (this.isInitiated() && this.fileInfo().shortDescription === ""){
+            return EagleConfig.getColor('graphWarning')
+        }
+
+        return ""
+    }, this);
+
+    getDetailedDescriptionBtnColor : ko.PureComputed<string> = ko.pureComputed(() => {
+        //this excludes graphs that have not been initiated by the user (eagle has an empty graph by default)
+        if (this.isInitiated() && this.fileInfo().detailedDescription === ""){
+            return EagleConfig.getColor('graphWarning')
+        }
+
+        return ""
+    }, this);
+
+    getGraphInfoBtnColor : ko.PureComputed<string> = ko.pureComputed(() => {
+        //this excludes graphs that have not been initiated by the user (eagle has an empty graph by default)
+        if (this.isInitiated() && this.fileInfo().detailedDescription === "" || this.fileInfo().name != "" && this.fileInfo().shortDescription === ""){
```

Comment:
Maybe you could simplify the logic here. My suggestion might not be exactly what you intended, I'm not sure.

```suggestion
        if (this.isInitiated() && (this.fileInfo().detailedDescription === "" || this.fileInfo().shortDescription === "")){
```